### ISSUE TITLE
Make liquibase compatible with Django 1.9

### ISSUE THREAD

**Issue by marcinn** [onjin/liquimigrate#4]
Liquimigrate does not work anymore with django 1.9:

```
TypeError: emit_pre_migrate_signal() takes exactly 3 arguments (4 given)
```

Possibly emit_pre_migrate_signal() signature has changed in django 1.9…

**Comment by onjin** [comment]
Was it fixed in #9 ?

**Comment by marcinn** [comment]
Yes, I think so.
